prow/tide/tide.go

```diff
@@ -813,6 +813,12 @@ func (c *Controller) mergePRs(sp subpool, prs []PullRequest) error {
 					// push to a specific branch.
 					log.WithError(err).Error("Merge failed: Branch needs to be configured to allow this robot to push.")
 					break
+				} else if _, ok = err.(github.MergeCommitsForbiddenError); ok {
```

Comment:
Do we have unit tests for the other scenarios?

Reply:
No